code/game/machinery/vending/vendor_types/crew/vehicle_crew.dm

```diff
@@ -170,44 +169,44 @@ GLOBAL_LIST_INIT(cm_vending_vehicle_crew_tank_spare, list(
-	list("TOW Launcher", 300, /obj/item/hardpoint/secondary/towlauncher, null, VENDOR_ITEM_REGULAR),
+	list("M56 Cupola", 0, /obj/item/hardpoint/secondary/m56cupola, null, VENDOR_ITEM_REGULAR),
+	list("LZR-N Flamer Unit", 0, /obj/item/hardpoint/secondary/small_flamer, null, VENDOR_ITEM_REGULAR),
+	list("TOW Launcher", 0, /obj/item/hardpoint/secondary/towlauncher, null, VENDOR_ITEM_REGULAR),
 
 	list("SECONDARY AMMUNITION", 0, null, null, null),
 	list("M92T Grenade Launcher Magazine", 50, /obj/item/ammo_magazine/hardpoint/tank_glauncher, null, VENDOR_ITEM_REGULAR),
-	list("M56 Cupola Magazine", 50, /obj/item/ammo_magazine/hardpoint/m56_cupola, null, VENDOR_ITEM_REGULAR),
+	list("M56 Cupola Magazine", 0, /obj/item/ammo_magazine/hardpoint/m56_cupola, null, VENDOR_ITEM_REGULAR),
 	list("LZR-N Flamer Unit Fuel Tank", 50, /obj/item/ammo_magazine/hardpoint/secondary_flamer, null, VENDOR_ITEM_REGULAR),
 	list("TOW Launcher Magazine", 50, /obj/item/ammo_magazine/hardpoint/towlauncher, null, VENDOR_ITEM_REGULAR),
 
 	list("SUPPORT MODULE", 0, null, null, null),
-	list("Artillery Module", 300, /obj/item/hardpoint/support/artillery_module, null, VENDOR_ITEM_REGULAR),
+	list("Artillery Module", 0, /obj/item/hardpoint/support/artillery_module, null, VENDOR_ITEM_REGULAR),
 	list("Integrated Weapons Sensor Array", 200, /obj/item/hardpoint/support/weapons_sensor, null, VENDOR_ITEM_REGULAR),
-	list("Overdrive Enhancer", 200, /obj/item/hardpoint/support/overdrive_enhancer, null, VENDOR_ITEM_REGULAR),
+	list("Overdrive Enhancer", 0, /obj/item/hardpoint/support/overdrive_enhancer, null, VENDOR_ITEM_REGULAR),
 
 	list("ARMOR", 0, null, null, null),
-	list("Ballistic Armor", 300, /obj/item/hardpoint/armor/ballistic, null, VENDOR_ITEM_REGULAR),
-	list("Caustic Armor", 300, /obj/item/hardpoint/armor/caustic, null, VENDOR_ITEM_REGULAR),
-	list("Concussive Armor", 300, /obj/item/hardpoint/armor/concussive, null, VENDOR_ITEM_REGULAR),
-	list("Paladin Armor", 300, /obj/item/hardpoint/armor/paladin, null, VENDOR_ITEM_REGULAR),
-	list("Snowplow", 200, /obj/item/hardpoint/armor/snowplow, null, VENDOR_ITEM_REGULAR),
+	list("Ballistic Armor", 0, /obj/item/hardpoint/armor/ballistic, null, VENDOR_ITEM_REGULAR),
+	list("Caustic Armor", 0, /obj/item/hardpoint/armor/caustic, null, VENDOR_ITEM_REGULAR),
+	list("Concussive Armor", 0, /obj/item/hardpoint/armor/concussive, null, VENDOR_ITEM_REGULAR),
+	list("Paladin Armor", 0, /obj/item/hardpoint/armor/paladin, null, VENDOR_ITEM_REGULAR),
+	list("Snowplow", 0, /obj/item/hardpoint/armor/snowplow, null, VENDOR_ITEM_REGULAR),
 
 	list("TREADS", 0, null, null, null),
-	list("Reinforced Treads", 200, /obj/item/hardpoint/locomotion/treads/robust, null, VENDOR_ITEM_REGULAR),
-	list("Treads", 200, /obj/item/hardpoint/locomotion/treads, null, VENDOR_ITEM_REGULAR)))
+	list("Reinforced Treads", 0, /obj/item/hardpoint/locomotion/treads/robust, null, VENDOR_ITEM_REGULAR),
```

Comment:
pretty sure you are not meant to be able to buy both . this would mean that you have 2 spares since you are already making the tank spawn with tires